app/models/concerns/procurement_validator.rb

```diff
@@ -166,6 +168,18 @@ def validate_mobilisation_and_tupe
       errors.add(:mobilisation_period, :not_valid_with_tupe) if (mobilisation_period_required && mobilisation_period < 4) && tupe == true
     end
 
+    def lot_number_in_range
+      lot_number_in_correct_range = if assessed_value < 7000000
+                                      true
+                                    elsif assessed_value >= 7000000 && assessed_value <= 50000000
```

Comment:
```suggestion
                                    elsif assessed_value <= 50000000
```
`assessed_value` will be >= 7000000 due to first condition

Reply:
Good spot, I've done that elsewhere as well 🤦